packages/app/src/cli/api/graphql/all_orgs.ts

```diff
@@ -5,7 +5,7 @@ export interface AllOrganizationsQuerySchemaOrganization {
   businessName: string
   website?: string
   betas: {
-    appUiDeployments?: boolean
+    cliTunnelAlternative?: boolean
```

Comment:
??

Reply:
@isaacroldan I've seen this flag pop up recently but I haven't seen it queried for consistently so this change brings consistency.